docs/dxp/7.x/en/installation-and-upgrades/installing-liferay-dxp-on-premises/installing-liferay-on-an-application-server/installing-liferay-on-jboss-eap.md

```diff
@@ -0,0 +1,350 @@
+
+In the `$JBOSS_HOME/bin/` folder, modify the standalone domain's configuration script file `standalone.conf` (`standalone.conf.bat` on Windows):
+
+* Set the file encoding to `UTF-8`
+* Set the user time zone to `GMT`
+* Set the preferred protocol stack
+* Increase the default amount of memory available.
+
+| **Important:** For DXP to work properly, the application server JVM must use the `GMT` time zone and `UTF-8` file encoding.
+
+Make the following edits as applicable to the respective operating system:
+
+**Windows:**
+
+1. Comment out the initial `JAVA_OPTS` assignment as demonstrated in the following line:
+
+    ```bash
+    rem set "JAVA_OPTS=-Xms1G -Xmx1G -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=2560m"
+    ```
+
+1. Add the following `JAVA_OPTS` assignment one line above the `:JAVA_OPTS_SET` line found at end of the file:
+
+    ```bash
+    set "JAVA_OPTS=%JAVA_OPTS% -Dfile.encoding=UTF-8 -Djava.net.preferIPv4Stack=true -Djboss.as.management.blocking.timeout=480 -Duser.timezone=GMT -Xms2560m -Xmx2560m -XX:MaxMetaspaceSize=768m"
+    ```
+
+**Unix:**
+
+1. Below the `if [ "x$JAVA_OPTS" = "x" ];` statement, replace this `JAVA_OPTS`
+    statement:
+
+    ```bash
+    JAVA_OPTS="-Xms1303m -Xmx1303m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=2560m -Djava.net.preferIPv4Stack=true"
+    ```
+
+    with this:
+
+    ```bash
+    JAVA_OPTS="-Djava.net.preferIPv4Stack=true"
+    ```
+
+1. Add the following statement to the bottom of the file:
+
+    ```bash
+    JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8 -Djava.net.preferIPv4Stack=true -Djboss.as.management.blocking.timeout=480 -Duser.timezone=GMT -Xms2560m -Xmx2560m -XX:MaxMetaspaceSize=512m"
+    ```
+
+On JDK 11, it's recommended to add this JVM argument to display four-digit years.
+
+```bash
+-Djava.locale.providers=JRE,COMPAT,CLDR
+```
+
+| **Note:** If using the IBM JDK with the JBoss server, complete some additional steps.
+
+1. Navigate to the `$JBOSS_HOME/modules/com/liferay/portal/main/module.xml` file and insert the following dependency within the `<dependencies>` element:
+
+     `<module name="ibm.jdk" />`
+
+1. Navigate to the `$JBOSS_HOME/modules/system/layers/base/sun/jdk/main/module.xml` file and insert the following path names inside the `<paths>...</paths>` element:
+
+```xml
+    <path name="com/sun/crypto" />
+    <path name="com/sun/crypto/provider" />
+    <path name="com/sun/image/codec/jpeg" />
+    <path name="com/sun/org/apache/xml/internal/resolver" />
+    <path name="com/sun/org/apache/xml/internal/resolver/tools" />
+```
+
+The added paths resolve issues with portal deployment exceptions and image uploading problems.
+
+**Checkpoint:**
+
+1. The file encoding, user time-zone, preferred protocol stack have been set in the `JAVA_OPTS` in the `standalone.conf.bat` file.
+
+1. The default amount of memory available has been increased.
+
+The prescribed script modifications are now complete for the DXP installation on JBoss.
+
+### Database Configuration
```

Comment:
Promote _Database Configuration_ and _Mail Configuration_ to level 2 headers as they can be done outside of the app server, in DXP.